### DISCUSSION TITLE
#[] is not working on me.

### DISCUSSION THREAD

**Discussion by mendylivium** [livewire/livewire#5902] [1. Help]
I tried to update to V3, I've noticed that **#[]** is not working on my proj., and I tried to make a new project and use Livewire V3 to test if its really not working, then I test and really its not working.

This is my test code
-Sender Component Code

```
class Sender extends Component
{

    public function send()
    {
        $this->dispatch('send')->to(Reciever::class);
    }

    public function render()
    {
        return view('livewire.sender');
    }
}
```

-Reciever Component Code

```
class Reciever extends Component
{

    #[On('send')]
    public function send()
    {
        dd('Received');
    }

    public function render()
    {
        return view('livewire.reciever');
    }
}
```

Main Blade

```
<body>
    <livewire:sender />
    <livewire:reciever />
</body>
```

**Comment by danie-ramdhani** [comment]
uh, works for me...

```php
// sender component 
public function send()
{
    $this->dispatch('sendToSiblingComponent')->to(Reciever::class);
}


// receiver component
use Livewire\Attributes\On;

#[On('sendToSiblingComponent')]
public function received()
{
    dd('ReceivedFromSender');
}
```

**Reply by mendylivium** [reply]
Thnak you for your sample code.

I just forgot to put "use Livewire\Attributes\On;"

**Comment by viliusvsx** [comment]
Do you get any error when trying to use attribute?

**Reply by mendylivium** [reply]
I dont see any errors, you can check the code here, I dont know what I have done wrong.

https://github.com/mendylivium/Livewire3_Sample

**Reply by mendylivium** [reply]
I've forgot to put "use Livewire\Attributes\On;" Hehe, now its working

But I'm not recieving any errors
